### PR TITLE
Added cookies prefrences simple card

### PR DESCRIPTION
**Pull request by chenchunaidu**

### CODE REVIEW

File: src/data/index.tsx [hunk 1]
```diff
@@ -19,7 +20,15 @@ export const data: Category[] = [
...
-    children: [hero, features, footer, testimonials, pricing, statistics],
+    children: [
+      hero,
+      features,
+      footer,
+      testimonials,
+      pricing,
+      statistics,
+      cookiePreferences,
+    ],
```
Comment (hauptrolle) [inline src/data/index.tsx:31]: I think that does not fit really into the Page  `Sectionts category`. I would move it into `Cards`

File: src/pages/templates/page-sections/cookiePreferences/simpleCookiePreference.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+export default function SimpleCookiePreference() {
+  return (
+    <Stack p="4" boxShadow="lg" m="4" borderRadius="sm">
+      <Stack direction="row" alignItems="center">
+        <Text fontWeight="semibold">Your Privary</Text>
+        <FcLock />
+      </Stack>
+
+      <Stack direction="row">
```
Comment (hauptrolle) [inline src/pages/templates/page-sections/cookiePreferences/simpleCookiePreference.tsx:13]: Maybe add some responsive behaviour here? So the buttons are below the text on small viewports

**Review by hauptrolle — CHANGES_REQUESTED** [review]
@chenchunaidu Thanks for your contribution. It should be ready to merge when u fix the 2 open comments 👍

File: src/data/index.tsx (hunk 1 above)
Comment (chenchunaidu) [reply src/data/index.tsx:31]: okay i will move it into cards

File: src/pages/templates/page-sections/cookiePreferences/simpleCookiePreference.tsx (hunk 2 above)
Comment (chenchunaidu) [reply src/pages/templates/page-sections/cookiePreferences/simpleCookiePreference.tsx:13]: i will check this

**Review by hauptrolle — APPROVED** [review]

**Comment by hauptrolle** [comment]
@all-contributors please add @chenchunaidu for code

**Pull request merged**